config.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	PollInterval         string    `json:"poll-interval"`
 	PollIntervalDuration time.Duration
 	MaxRequestsPerQuery  int64 `json:"max-requests-per-query"`
+	MaxResponseSize      int64 `json:"max-service-response-size"`
```

Comment:
The name should probably be the same between the struct and the json tag (`MaxServiceResponseSize`)